### ISSUE TITLE
The included notification shell script fails

### ISSUE THREAD

**Issue by david-christiansen** [matterhorn-chat/matterhorn#401]
When getting a mention from Matterhorn, I get the following in the log:
```
Program: /home/dtc/bin/mh-notify
Arguments: ["2","USER","MESSAGE"]
Exit code: ExitFailure 1
Stdout:

Stderr:
Unknown urgency  specified. Known urgency levels: low, normal, critical.
```

Turning on `set -x` yields:
```Program: /home/dtc/bin/mh-notify
Arguments: ["2","USER","TEXT"]
Exit code: ExitFailure 1
Stdout:

Stderr:
+ mentioned=2
+ sender=USER
+ message='TEXT'
+ notify_URGENCIES='normal normal'
+ notify_CATEGORY=im.received
+ notify_HEAD='Matterhorn message from cygnus'
+ notify_BODY='this script needs work.'
++ getUrgency
++ getUrgencyHelper normal normal
++ shift 2
++ echo ''
+ test '' = none
++ getUrgency
++ getUrgencyHelper normal normal
++ shift 2
++ echo ''
+ notify-send -u '' -c im.received 'Matterhorn message from USER' 'TEXT'
Unknown urgency  specified. Known urgency levels: low, normal, critical.
```

**Comment by jtdaugherty** [comment]
Fixed as of `fcb1310`. Thanks!

**Comment by david-christiansen** [comment]
Thank you!
